### ISSUE TITLE
How this tool gives Response in red or in green?

### ISSUE THREAD

**Issue by Ingsuifon** [frigus02/RESTer#121]
I'm a new hand studying http-server recently, I know that it returns different HTTP Status Code such as 200 and 404 according to different cases, but I think in programs they are the same so I want to know how does your tool distinguish them?

**Comment by frigus02** [comment]
Hi @Ingsuifon, you're right. The HTTP status code is just a number returned by the server as part of the response.

The code in RESTer is a bit convoluted. It uses a few browser extension APIs to get around security limitations of normal websites. This makes it quite hard to follow. But essentially, RESTer just uses [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) to make an HTTP request and then reads the [`status` from the response](https://developer.mozilla.org/en-US/docs/Web/API/Response/status).

https://github.com/frigus02/RESTer/blob/d53cbb9cfc1c810be989cac5072c4dc9e1d06011/src/site/elements/data/scripts/request.js#L276

This results in a number, which would be in the rage of 100 - 599. By convention [status codes are divided into 5 groups](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status). I decided to show this in RESTer by applying a different color per group:

https://github.com/frigus02/RESTer/blob/d53cbb9cfc1c810be989cac5072c4dc9e1d06011/src/site/elements/pages/rester-page-request.js#L746-L756

Does this make sense? Did I understand your question correctly?

**Comment by Ingsuifon** [comment]
Thank you very much. I haven't studied javaScript so I temporarily cann't read your code. But I understand your design concept. Thank you.

**Comment by frigus02** [comment]
Great. I'm going to close this issue. But please feel free to re-open or open a new one if you have further questions.
